API/Services/Implementations/JwtService.cs

```diff
@@ -0,0 +1,156 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using API.Authorization;
+using API.Configurations;
+using API.Services.Interfaces;
+using Database.Entities;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+namespace API.Services.Implementations;
+
+public class JwtService(
+    ILogger<JwtService> logger,
+    IOptions<JwtConfiguration> jwtConfiguration
+) : IJwtService
+{
+    private readonly JwtConfiguration _jwtConfiguration = jwtConfiguration.Value;
+
+    public int AccessDurationSeconds => 3600;
+    public int RefreshDurationSeconds => 1_209_600;
```

Comment:
For clarity either comment the total duration or use `(int) TimeSpan.FromX().TotalSeconds`

Reply:
These fields are documented in the interface `IJwtService`

```cs
    /// <summary>
    /// Lifetime of an access token in seconds
    /// </summary>
    public int AccessDurationSeconds { get; }

    /// <summary>
    /// Lifetime of a refresh token in seconds
    /// </summary>
    public int RefreshDurationSeconds { get; }
```